internal/performance/create_job_run_test.go

```diff
@@ -0,0 +1,90 @@
+package performance
```

Comment:
We write tests in an external package, like `performance_test`. This forces you to consider the API that other packages consume by not allowing you access to unexported functions and attributes. (Treat unexproted variables in a package like private variables in other languages.) I'm leaning towards treating benchmarks the same way.